test/test_rb.py

```diff
@@ -0,0 +1,237 @@
+            self.assertEqual(
+                matrix_equal(Clifford(qc).to_matrix(), np.identity(2 ** num_qubits)),
+                True,
+                "Clifford sequence doesn't result in the identity matrix.",
+            )
+
+    def is_identity_transpiled(self, transpiled_circuits: list):
+        """Standard randomized benchmarking test - Identity check for the transpiled circuits.
+            Using
+        Args:
+            transpiled_circuits (list): list of the circuits which we want to check
+        """
+        for qc in transpiled_circuits:
+            num_qubits = qc.num_qubits
+            qc.remove_final_measurements()
+            # Checking if the matrix representation is the identity matrix
+            self.assertAlmostEqual(
+                process_fidelity(Clifford(qc).to_matrix(), np.identity(2 ** num_qubits)),
+                1,
+                "Transpiled circuit doesn't result in the identity operator.",
+            )
+
+    def validate_metadata(self, circuits: list, exp_attributes: dict):
+        """
+        Validate the fields in "metadata" for the experiment.
+        Args:
+            circuits (list): A list containing quantum circuits
+            exp_attributes (dict): A dictionary with the experiment variable ands values
+        """
+        for ind, qc in enumerate(circuits):
+            self.assertEqual(
+                qc.metadata["xval"],
+                exp_attributes["lengths"][ind],
+                "The number of gates in the experiment metadata doesn't match to the one provided.",
+            )
+            self.assertEqual(
+                qc.metadata["qubits"],
+                tuple(exp_attributes["qubits"]),
+                "The qubits indices in the experiment metadata doesn't match to the one provided.",
+            )
+
+    def validate_circuit_data(
+        self, experiment: qe.experiment_data.ExperimentData, exp_attributes: dict
+    ):
+        """
+        Validate that the metadata of the experiment after it had run matches the one provided.
+        Args:
+
+            experiment(qiskit_experiments.experiment_data.ExperimentData): The experiment
+            data and results after it run.
+            exp_attributes (dict): A dictionary with the experiment variable ands values
+        """
+        for ind, data in enumerate(experiment.data):
+            experiment_information = data["metadata"]
+            self.assertEqual(
+                experiment_information["xdata"],
+                exp_attributes["lengths"][ind],
+                "The number of gates in the experiment doesn't match to the one in the metadata.",
+            )
+            self.assertEqual(
+                experiment_information["qubits"],
+                exp_attributes["qubits"],
+                "The qubits indices in the experiment doesn't match to the one in the metadata.",
+            )
+
+    def _exp_data_prpeties(self):
```

Comment:
using "properties" is much better than "prpeties" - readability is important, and the bits we save by skipping letters are not worth it.

Reply:
Changed it to "properties"